static/js/popovers.js

```diff
@@ -42,14 +42,15 @@ function load_medium_avatar(user_email) {
 }
 
 function user_last_seen_time_status(user_id) {
-    if (presence.get_status(user_id) === "active") {
+    var status = presence.get_status(user_id);
+    if (status === "active") {
         return i18n.t("Active now");
     }
-
-    if (people.get_person_from_user_id(user_id).is_bot) {
```

Comment:
@showell Maybe later we can show the icon of a bot instead of no status indicator once we fix the issue with `is_bot`